packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -301,6 +305,14 @@ class Tooltip extends ThemePropertyMixin(ElementMixin(PolymerElement)) {
     if (this._autoOpened) {
       this._close(true);
     }
+    this.__connected = false;
+  }
+
+  /** @protected */
+  connectedCallback() {
```

Comment:
nit: let's place `connectedCallback()` before `disconnectedCallback()` as we generally do.

Reply:
Done